cmd/docker/docker.go

```diff
@@ -13,6 +13,9 @@ import (
 	cliconfig "github.com/docker/cli/cli/config"
 	"github.com/docker/cli/cli/debug"
 	cliflags "github.com/docker/cli/cli/flags"
+	"github.com/docker/cli/project"
+	projectImpl "github.com/docker/cli/project/impl"
+	projectutil "github.com/docker/cli/project/util"
```

Comment:
I don't really like those 2 packages. Let's just put the implementation in `project` package ; and find a better name for `util` (depending on what it does) and/or moving the behaviour in this package elsewhere.